python_ta/contracts/__init__.py

```diff
@@ -430,7 +430,10 @@ def parse_assertions(obj: Any, parse_token: str = "Precondition") -> List[str]:
        line is of the form "- <cond>". Each line is considered a separate condition.
        The lines can be separated by blank lines, but no other text.
     """
-    docstring = getattr(obj, "__doc__") or ""
+    if isinstance(obj, str):
```

Comment:
The only problem that I'm running into is that the original function takes `node.__doc__`, but I want `node.doc`.

Reply:
@peelstnac ah great point. Still, I'd rather try to access both variables here to support passing in astroid nodes, rather than passing in a string directly. Something like `getattr(obj, "__doc__") or getattr(obj, "doc") or ""`.